storage-operator/deploy/role.yaml

```diff
@@ -10,14 +10,21 @@ rules:
   - pods
   - nodes
   - services
+  - services/finalizers
   - endpoints
   - persistentvolumes
   - persistentvolumeclaims
   - events
   - configmaps
   - secrets
   verbs:
-  - '*'
+  - create
```

Comment:
Seems like this started to tackle https://github.com/scality/metalk8s/issues/2084 as a side effet :slightly_smiling_face: